sherpa/astro/xspec/src/_xspec.cc

```diff
@@ -284,6 +284,14 @@ void C_reflct(const double* energy, int nFlux, const double* params, int spectru
 void C_simpl(const double* energy, int nFlux, const double* params, int spectrumNumber, double* flux, double* fluxError, const char* initStr);
 void C_zashift(const double* energy, int nFlux, const double* params, int spectrumNumber, double* flux, double* fluxError, const char* initStr);
 void C_zmshift(const double* energy, int nFlux, const double* params, int spectrumNumber, double* flux, double* fluxError, const char* initStr);
+
+// Models from 12.9.1
+//
+//
+#ifdef XSPEC_12_9_1
```

Comment:
As shown in #394, we can actually remove the declarations of the C-interface models (those whose name begins with `C_`) as we can use just include XSPEC's `funcWrappers.h` (well, need to set up the include path, but that's not a lot of work).

That's not to say that there might need to be some level of conditional includes here, but hopefully only in rare cases (we need to declare the models using the Fortran-style interface; or, rather, I couldn't work out how to avoid this, but hopefully  most new models aren't written to this API)

Reply:
Since you mention it, how much of #394 is taking advantage of stuff introduced in 12.9.1 and not present in 12.9.0?

Reply:
At present #394 only binds to routines present in the `xsFortran` header of 12.9.0.

Reply:
I'm leaving this out of this PR... I have some questions regarding #394 but that's where we might remove some code.